manage.py

```diff
@@ -0,0 +1,30 @@
+# # SQLALCHEMY_DATABASE_URI = 'postgres://localhost/main'
```

Comment:
I typically try to remove any commented out code if it's not going to be used later.

Reply:
I'll remove it